db/schema.rb

```diff
@@ -105,16 +105,15 @@
     t.string   "peer_industry"
     t.string   "current_goal"
     t.text     "top_3_interests",                        default: [],                 array: true
-    t.boolean  "live_in_detroit",                        default: true
     t.boolean  "waitlist",                               default: true
     t.boolean  "is_assigned_peer_group",                 default: false
     t.integer  "mentor_times",                           default: 1
     t.integer  "mentor_limit",                           default: 1
     t.boolean  "is_participating_this_month"
     t.string   "image_url"
     t.integer  "location_id"
-    t.string   "linkedin_url"
     t.string   "zip_code",                    limit: 10
+    t.string   "linkedin_url"
```

Comment:
@rfjackson11, I'm confused about why this addition is in this PR. Is this related to public user profiles?